tests/functional/fx_firstrun_v1_sign_up.js

```diff
@@ -58,11 +58,12 @@ define([
           return FunctionalHelpers.fillOutSignUp(self, email, PASSWORD);
         })
 
+        .findByCssSelector('#fxa-confirm-header')
```

Comment:
@vladikoff - the problem was here, we were checking for `fxaccounts:login` before the screen transition, so it wasn't yet fired.